data.sql

```diff
@@ -10,4 +10,22 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', 'January 17, 2021', 1, false, 15.04 );
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
-VALUES ('Devimon', 'May 12, 2017', 5, true, 11 );
\ No newline at end of file
+VALUES ('Devimon', 'May 12, 2017', 5, true, 11 );
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', 'February 8, 2020', 0, false, -11 );
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon', 'November 15, 2022', 2, true, -5.7 );
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Squirtle', 'April 2, 1993', 3, false, -12.13 );
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Angemon', 'June 12, 2005', 1, true, -45 );
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Boarmon', 'June 7, 2005', 7, true, -20.4 );
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
```

Comment:
Please change the values of `weight_kg` to positive as it's asked to change by the previous reviewer. Thanks!